scripts/attempt-backport.js

```diff
@@ -206,12 +223,22 @@ function attemptBackport (options, version, isLTS, cb) {
     const cp = wrapCP('git', ['am'], { stdio: 'pipe' }, function done () {
       // Success!
       if (isLTS) {
-        updatePrWithLabels(options, [`lts-watch-v${version}.x`])
-      }// else {
+        fetchExistingThenUpdatePr(options, [`lts-watch-v${version}.x`])
+      } else {
         // TODO(Fishrock123): Re-enable this, but do a check first
         // to make sure the label was set by the bot only.
```

Comment:
Yeah, fixed in https://github.com/nodejs/github-bot/commit/5688b42be393821fa5cf8f0e5b169cd3cd6526cf